src/taipy/rest/api/resources/job.py

```diff
@@ -34,12 +33,23 @@ class JobResource(Resource):
       tags:
         - api
       summary: Get a job
-      description: Get a single job by ID
+      description: |
+        Return a single job by job_id. If the job does not exist, a 404 error is returned.
```

Comment:
'Return a single job by job_id.'
->
'The job whose identifier is \*job_id\* is returned.'

Reply:
I think the meaning of 'Return a single job by job_id.' is more straightforward.